InfiniteLoopBookProject/src/com/st/lms/model/Book.java

```diff
@@ -0,0 +1,38 @@
+package com.st.lms.model;
+
+public class Book extends Entry{
+	private int authorID, publisherID;

```

Comment:
I see that you haven't actually overridden the equals() and hashcode() in the models/pojos, which would be great if you do so. because it's a good practice and it actually helps a lot if you need to compare values...